packages/third-parties/pulse/test/pulse-define.integration.spec.ts

```
@@ -81,7 +82,7 @@ describe("Pulse integration", () => {
     afterAll(() => TestContainersMongo.reset());
 
     it("should not have job definitions", () => {
-      const pulse = PlatformTest.injector.get(PulseService)!;
+      const pulse = PlatformTest.injector.get(Pulse)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)